src/core/Akka.FSharp/FsApi.fs

```diff
@@ -424,6 +429,31 @@ module Spawn =
         spawnOpt actorFactory name f []
 
     /// <summary>
+    /// Spawns an actor using specified actor quotation, with custom spawn option settings.
+    /// The actor can only be used locally. 
+    /// </summary>
+    /// <param name="actorFactory">Either actor system or parent actor</param>
+    /// <param name="name">Name of spawned child actor</param>
+    /// <param name="f">Used to create a new instance of the actor</param>
+    /// <param name="options">List of options used to configure actor creation</param>
+    let spawnObjOpt<'Actor when 'Actor :> ActorBase> (actorFactory : ActorRefFactory) (name : string) (f : Quotations.Expr<(unit -> 'Actor)>)
```

Comment:
In this case `<'Actor when 'Actor :> ActorBase>` is unnecessary. Equivalent function signature is:
`let spawnObjOpt (actorFactory : ActorRefFactory) (name : string) (f : Quotations.Expr<(unit -> #ActorBase)>)`